src/components/Navbar/Navbar.styled.tsx

```diff
@@ -9,24 +9,29 @@ export const StyledNavbar = styled.nav`
   height: 64px;
   padding: 0 1em;
 
-  background-color: ${({ theme }) => theme.color.primary};
-  color: ${({ theme }) => theme.color.white};
+  background-color: ${({ theme }) => theme.color.white};
+  color: ${({ theme }) => theme.color.primary};
 
   display: grid;
-  grid-template-columns: 20% 1fr 10%;
+  grid-template-columns: 15em 1fr 10%;
   align-items: center;
+  border-bottom: 1px solid ${({ theme }) => theme.color.grey};
 `;
 
 export const StyledNavbarSection = styled.div`
   display: flex;
   align-items: center;
   justify-content: start;
 
-  gap: 2em;
+  gap: 1em;
 
   img {
     max-width: 100%;
```

Comment:
We can move this to global style, to make it responsive and fit within their containers all images should have max width 100%